Library/Formula/mhash.rb

```diff
@@ -1,14 +1,16 @@
-require 'formula'
-
 class Mhash < Formula
   desc "Uniform interface to a large number of hash algorithms"
-  homepage 'http://mhash.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/project/mhash/mhash/0.9.9.9/mhash-0.9.9.9.tar.gz'
-  sha1 'c898de5ea60d9e0873a1b73caa031bb1b5797c03'
+  homepage "http://mhash.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/mhash/mhash/0.9.9.9/mhash-0.9.9.9.tar.gz"
+  sha256 "3dcad09a63b6f1f634e64168dd398e9feb9925560f9b671ce52283a79604d13e"
 
   def install
     system "./configure", "--disable-debug", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "otool -L #{prefix}/lib/libmhash.2.dylib | grep libSystem"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!